Standards/scs-XXXX-vN-decision-record-template.md

```diff
@@ -0,0 +1,41 @@
+---
+title: _Descriptive title_
```

Comment:
I think @markus-hentsch somewhere remarked that there were no parts in italics/emphasized, but this is a counter example.